### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by Knofbath**

### CODE REVIEW

File: binary_translations.pm [hunk 1]
```diff
@@ -253,50 +253,60 @@ sub data {
...
+        "輸送船 x {0}" => {},                             #"Transports x {0}" 
+        "ロック済艦娘を" => {},                            #"Ship Girl is locked" literal "Locked ship daughter" 
+        "艦隊司令部情報" => {},                            #"Fleet Headquarters intelligence"
+        "艦隊切替" => {},                                 #"Switch fleet"
+        "艦隊数" => {},                                   #"Fleet number"
+        "ボタンで戦略コマンド" => {},                       #"Press button for Strategy Command"
+        "[000000][0055aa]輸送船[-]の配備について" => {},    #"Transports Deployed" - My context may be off, but OC this is how I am reading it.
+        #                                                  Depriciated "About deployment of Transports" alternate "About Transport deployment"
+        "工廠[-]での輸送船建造や、" => {},                  #"Factory[-]Building a Transport ship at," alternate "Building a Transport ship at the Factory" color codes missing from string would go around Factory
+        "[66ccff]全艦種出撃可能[-]です。"=>{},              #"All classes of ships may sortie."
```
Comment (Knofbath) [inline binary_translations.pm:302]: I'm not backing down on this Type vs Class thing.

https://en.wikipedia.org/wiki/List_of_types_of_naval_vessels

Comment (miyagami) [reply binary_translations.pm:302]: Sort by Class Link: http://kancolle.wikia.com/wiki/Ship_Class
http://en.kancollewiki.net/wiki/Ship_Class

http://kancolle.wikia.com/wiki/Ship


http://en.kancollewiki.net/wiki/Glossary#Class_classification


I should point out, that this is where you are making your mistake. Like all uneducated misconceptions, there is some small amount of truth to what you are saying. But, the truth stops as soon as light is shown.
It is true, a Destroyer is a type of ship. Just as it is true, an aircraft carrier, or a light cruiser is a type of ship.
But, a DD and a DE are different classes of ship inside the destroyer type.
A CVL, CV, CVB are some of the classes of ships inside of the aircraft carrier type.
And, let us not forget, CL, CLT, CT, are some of the classes inside the light cruiser type.

So, as the game breaks them down to the Classes within the different Types, as shown here:

We are using Class.

And, stop citing Wikipedia as your only sources. Not only is it considered not acceptable as an authoritative source by Universities of good reputation, it does not even consider itself to be a source of verifiable information.

Pick another mound to spill your blood on. This one is over.

Comment (Knofbath) [reply binary_translations.pm]: The Ship Class link shows the various classes of ship: http://kancolle.wikia.com/wiki/Ship_Class

Destroyer
- Kamikaze-class
- Mutsuki-class
- Fubuki-class
- Ayanami-class
- Akatsuki-class
- Hatsuharu-class
- Shiratsuyu-class
- Asashio-class
- Kagerou-class
- Yuugumo-class
- Akizuki-class
- Shimakaze (unique ship with no sister ships thus not a class)

Destoyer is the Type, the Class is named after the lead ship of that class.

> A ship class is a group of ships of a similar design. This is distinct from a ship type, which might reflect a similarity of tonnage or intended use. For example, USS Carl Vinson is a nuclear aircraft carrier (ship type) of the Nimitz class (ship class). ... Ships of the same class may be referred to as sister ships.

**Comment by miyagami** [comment]
Thank you for catching the spelling mistake.

**Comment by wchristian** [comment]
I've split this one up into #22 and #23, where further work will need to continue. This PR is thus closed.

**Pull request closed without merging**